packages/.template/package.json

```diff
@@ -23,7 +23,7 @@
     "watch": "yarn build --watch"
   },
   "dependencies": {
-    "@zendeskgarden/css-variables": "^5.0.0"
+    "@zendeskgarden/css-variables": "^5.0.1"
```

Comment:
Rather than manually keeping this up-to-date, should we not include it?

In react-components we have lerna add it during the creation process, but that is pretty tedious.

Reply:
from what I'm seeing in #85 it looks like Renovate detects this file – which is good for me. The PR is just simplifying the churn required by initial renovate.